tests/Jira/ApiTest.php

```diff
@@ -0,0 +1,23 @@
+<?php
+
+namespace Tests\chobie\Jira;
```

Comment:
Are you sure these tests are auto-loaded? The namespace for tests in `autoload-dev` part of `composer.json` says `chobie\\Tests\\` namespace.

Probably need to patch it in there.